src/services/users.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const { getNewId } = require('../utils/helpers');
+
+let users = [];
+
+const getAll = () => {
+  return users;
+};
+
+const getById = (userId) => {
+  const foundUser = users.find(({ id }) => id === +userId);
+
+  return foundUser;
+};
+
+const create = (userBody) => {
+  const newUser = {
+    id: getNewId(users),
+    ...userBody,
```

Comment:
What if `userBody` contains some unneeded fields? Because of this, user can push any gibberish into your data. Make sure to filter your input body, and only save the data required by documentation.